include/main_functions_home.h

```diff
@@ -22,22 +22,25 @@
  * parameter: none 
  * return_values: none
  */
-void help_page();
+void help_page(bool *color);
 
 /* 
  * fileexists: check if file with name, given by string, exists
  * parameter: string of file name, potential target for editing
  * return: bool, TRUE means exists, FALSE means not
  */
-bool fileexists(const char* filename);
+bool fileexists(const char *filename);
 
 /*
  * change_mode: helper for main_page, determine input mode
- * parameter: command line input from main_page
+ * parameter:
+ *		- arg: command line input from main_page
+ *		- color: bool for color of error message (false for no color, true for color)
+ *
  * return: number indicating mode
  * 			1 means quiet, 2 verbose, 3 interactive
  */
-int change_mode(char* arg);
+int change_mode(char *arg, bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!